home/ivp-planning/keyboard-shortcuts.md

```diff
@@ -23,29 +30,43 @@ Go to "[Moving the camera](getting-started/moving-the-camera.md#moving-the-camer
 | **Right click + E**               | Up         | Moving the 3D camera up        |
 | **Right click + Q**               | Down       | Moving the 3D camera down      |
 
-## **Machine interaction**
+## **Paths**
+
+| Shortcut       | Input Name   | Function                             |
+| -------------- | ------------ | ------------------------------------ |
+| **P**          | Path Mode    | Activate Path mode                   |
+| **V**          | Regular Mode | Activate regular mode                |
+| **Left SHIFT** | Snapping     | Change angle only by [set](../ivp-planning/settings/global-settings.md#the-options-explained) degrees |
+
+## **Object interaction**
 
-| Shortcut                      | Input Name   | Function                        |
-| ----------------------------- | ------------ | ------------------------------- |
-| **F**                         | Focus Object | Zoom in to the selected machine |
-| **E**                         | Transform    | Activate 3D-transform tools     |
-| **R**                         | Rotate       | Activate 3D-rotate tools        |
-| **T**                         | Scale        | Activate 3D-scale tools         |
-| **Delete**                    | Delete       | Delete the selected machines    |
-| **Left STRG + LeftMouseDrag** | Box select   | Box selections for machines     |
-| **Left STRG + C**             | Copy         | Copy the selected machines      |
+| Shortcut                      | Input Name    | Function                                     |
+| ----------------------------- | ------------- | -------------------------------------------- | 
+| **F**                         | Focus Object  | Zoom in to the selected machine              | 
+| **W**                         | 3d Move       | Activate 3D-transform tools                  | 
+| **E**                         | Rotate        | Activate 3D-rotate tools                     |
+| **R**                         | Scale         | Activate 3D-scale tools                      |
+| **Delete**                    | Delete        | Delete the selected objects                  |
+| **Left CTRL + LeftMouseDrag** | Box select    | Box selections for objects (only in 2D view) |
+| **Left CTRL + C**             | Copy          | Copy the selected object                     |
+| **F7**                        | Duplicate     | Copy the selected object multiple times      |
```

Comment:
This is named "Multi-Copy" now

Reply:
was unsure whether to use this as its not mentioned anywhere else like that (would have changed when reworking the issue overall)
changed accordingly.